daemon/cmd/daemon_main.go

```diff
@@ -1654,7 +1654,7 @@ type daemonParams struct {
 	APILimiterSet        *rate.APILimiterSet
 	AuthManager          *auth.AuthManager
 	Settings             cellSettings
-	HealthV2Provider     healthTypes.Provider
+	HealthV2Provider     types.Provider
```

Comment:
having a import named "types" is always a bit annoying in a big file like daemon_main.go 😄